versioned/interface.ts

```diff
@@ -678,6 +696,37 @@ abstract class Versioned {
       ],
     };
   }
+
+  static update_metadata(version: version): {
+    values: { [key: string]: string };
+    fields: {
+      field: string;
+      label: string;
+      path: string;
+      placeholder: string;
+      validate: (p: string) => string | undefined;
+    }[];
+  } {
+    if (!hasTzip27Support(version)) {
+      return { fields: [], values: {} };
+    }
+
+    return {
+      values: {
+        tzip16_metadata: "",
+      },
+      fields: [
+        {
+          field: "tzip16_metadata",
+          label: "Metadata in TZIP16",
```

Comment:
Same than before, is putting the Tzip number worth it? 
If it this, then let's do like before and put it between parenthesis

Reply:
metadata is too common. We also have metadata with lamdba. There are users asking me before so I think it would be clear to add it.